internal/util/conn_pool_test.go

```diff
@@ -74,6 +74,7 @@ func (cp *ConnPool) fakeGet(monitors, user, keyfile string) (*rados.Conn, string
 }
 
 func TestConnPool(t *testing.T) {
+	t.Parallel()
```

Comment:
These tests are probably not suitable to run in parallel. After making modifications, try running `go test -v ./internal/util` and see if it fails for you too:
```
=== CONT  TestConnPool/fakeGet
    conn_pool_test.go:97: failed to get connection: could not open keyfile /tmp/conn_utils.keyfile: open /tmp/conn_utils.keyfile: no such file or directory
    conn_pool_test.go:104: there is more than a single conn in cp.conns: 0
    conn_pool_test.go:110: getting the conn from cp.conns failed
--- FAIL: TestConnPool (0.00s)
    --- FAIL: TestConnPool/fakeGet (0.00s)
```

Reply:
sure